policies/trustee_roles_and_responsibilities.md

```diff
@@ -0,0 +1,127 @@
+* their commitment to the organisation’s work
+* the skills and experience they bring.
+
+### Other key characteristics - Trustees should
+
+* understand the responsibilities they are taking on (and should be prepared to receive training if they don’t)
+* be prepared to commit the time needed to play an active role
+* be absolutely trustworthy.
+
+## Regular tasks for all trustees include
+
+* Active participation at regular board meetings (2 hrs/qtr)
+* Actioning tasks arising from regular board meetings
+  * These can be wide ranging and trustees should actively volunteer to take on tasks that they feel they are suited to carrying out. We aim to share the workload of tasks as much as we can.
+* Regular participation in discussions on Slack
+  * This is one of the main channels of Trustee discussion and people should be prepared to accept notifications of urgent discussions at all times and check in to the main discussion thread every few days
+  * Slack is also used to make urgent decisions that cannot wait until the next board meeting
+* Be an ambassador for Farset Labs and it’s values (internally and externally)
+* Management of communications channels (email, facebook, twitter)
+* Co-ordinating the [Annual General Meeting](../runbooks/rituals/annual_general_meeting.md)
+* Contributing to the Annual Trustees Report for submission with our annual accounts
+
+### Specific tasks and responsibilities of Secretary, Treasurer & Chair
+
+_Note - these are functional roles only and do not confer any additional authority_
+
+* [Chair](https://www.diycommitteeguide.org/code/principle/what-role-chairperson)
+  * Representing Farset Labs in an official capacity when necessary
+  * Chairing the Annual General Meeting and Trustees Meetings
+  * Monitoring and Validating that decisions taken at meetings are implemented
+* [Secretary](https://www.diycommitteeguide.org/code/principle/secretary)
+  * Setting agenda and recording minutes from board and members meetings
+  * Ensuring we meet our legal obligations with the Charities Commission for Northern Ireland and Companies House
+  * Draft agendas working with the other trustees and chair
+    – Take minutes in meetings or organise cover where necessary
+    – Receive correspondence and organise responses in a timely fashion
+    – Keep the registers of Trustees up to date with Companies House and the Charities Commission for Northern Ireland
+* [Treasurer](https://www.diycommitteeguide.org/code/principle/treasurer)
+  * Coordinating and monitoring budgets and managing finances
+  * Regularly reconciling our accounting balances
+  * Liaising with our accountant
+  * Ensuring we meet our financial obligations with Companies House and HMRC and that all submission deadlines are met
+    – Reminding members to consider the financial implications of decisions they are taking and not to take decisions unless they have sufficient information in front of them
```

Comment:
Possibly worth refactoring to clarify that this means financial implications _for farset_ rather than personally